src/ContinuousEnergyNeutron.tex

```diff
@@ -1200,3 +1272,77 @@ \subsubsection{\textsf{UNR} Block}\label{sec:UNRBlock}
 \end{XSSTable}
 \end{ThreePartTable}
+
+\subsubsection{\textsf{PTYPE} Block}\label{sec:PTYPEBlock}
+
+The \Block{PTYPE} is the first of the particle production blocks used for neutron and charged particle production data. These particle production blocks are given only if the number of particles \var{NTYPE} = \nxs{7} is different from zero. If the \Block{PTYPE} is present, the \Block{PTYPE} starts at the index \var{LTYPE} = \jxs{30}.
+
+{\color{red}\textbf{Question:} The particle type can be neutron, when is this actually used?}
+
+The \Block{PTYPE} gives a list of particle types for which particle production data is available. This includes cross section data (given in the \Block{SIGH}), angular distribution data (given in the \Block{ANDH}) and secondary particle energy distribution data (given in the \Block{DLWH}).
+
+The format of the \Block{PTYPE} is given in \Tableref{tab:PTYPEBlock}.
+
+\begin{BlockTable}{PTYPE}
+  \var{LTYPE}               & \var{IP}$_{1}$         & First particle type \\
+  \var{LTYPE}+1             & \var{IP}$_{2}$         & Second particle type \\
+  \ldots \\
+  \var{LTYPE}+\var{NTYPE}-1 & \var{IP}$_{\var{NMT}}$ & Last particle type
+  \label{tab:PTYPEBlock}
+\end{BlockTable}
+
+$\var{IP}_{1},\ldots,\var{IP}_{\var{NMT}}$ are particle identifiers given as follows:
+\begin{itemize}
+  \item $\var{IP} = 1$ for neutrons
+  \item $\var{IP} = 9$ for protons
+  \item $\var{IP} = 31$ for deuterons
+  \item $\var{IP} = 32$ for tritons
+  \item $\var{IP} = 33$ for helions
+  \item $\var{IP} = 34$ for aphas
+\end{itemize}
+
+\subsubsection{\textsf{NTRO} Block}\label{sec:NTROBlock}
+
+The \Block{NTRO} gives the number of reactions for each of the particle types defined in the \Block{PTYPE}. If the \Block{NTRO} is present, the \Block{NTRO} starts at the index \var{LTYPE} = \jxs{31}.
+
+The format of the \Block{NTRO} is given in \Tableref{tab:NTROBlock}.
+
+\begin{BlockTable}{NTRO}
+  \var{LTYPE}               & \var{NP}$_{1}$         & Number of reactions producing the first particle type \\
+  \var{LTYPE}+1             & \var{NP}$_{2}$         & Number of reactions producing the second particle type \\
+  \ldots \\
+  \var{LTYPE}+\var{NTYPE}-1 & \var{NP}$_{\var{NMT}}$ & Number of reactions producing the last particle type
+  \label{tab:NTROBlock}
+\end{BlockTable}
+
+\subsubsection{\textsf{IXS} Block}\label{sec:IXSBlock}
+
+The \Block{IXS} gives 10 particle production locators for each of the particle types defined in the \Block{PTYPE}. If the \Block{IXS} is present, the \Block{IXS} starts at the index \var{NEXT} = \jxs{32}. The \Block{IXS} serves a similar function as the \hyperref[sec:JXSContinuousEnergyNeutron]{JXS Array}, in that it provides the locators to specific blocks of data, as laid out in \Tableref{tab:IXSBlock}.
+
+\begin{BlockTable}{IXS array for particle type $j$, with $\var{LTYPE}=\var{NEXT}+10\var{NTYPE}(j-1)$}
+  \var{LTYPE}     & \var{HPD}    & Location of the total particle production and heating data \\
```

Comment:
It's a little confusing that you use the same variable, `LTYPE`, here in the `NTRO` Block as you do in the `PTYPE` Block. You do redefine what that is, but it caught me off guard a bit. A different name would be more clear.